### DISCUSSION TITLE
What are recommended AWS system requirements for polygon edge network?

### DISCUSSION THREAD

**Discussion by vineetaparodkar** [0xPolygon/polygon-edge#801] [Q&A]
1. What is the minimum and recommended AWS system requirement for 4 nodes validator network build using polygon edge?

2. Polygon edge docs recommends below configs per node [https://wiki.polygon.technology/docs/edge/validator-hosting]
CPU - 2Cores, RAM- 2GB, Disk- 10/30 GB partioning.
So for above configs are there any performance reports for below influencing factors?
   - Number of JSON-RPC queries
   - Size of the blockchain state
   - Block gas limit
   - Block time

3. What is recommended host configuration for AWS instance size, File descriptor limit? Are there any other AWS configuration that must be considered?

4. What is recommended blockchain configuration for Block time, Block gas limit? Are there any other blockchain configuration that must be considered when creating a genesis?

**Comment by laviniat1996** [comment]
Hi @vineetaparodkar ,

1. We do not have one yet.
2. Those are minimum requirements only and you can adapt them to your needs. We have performance reports here: https://wiki.polygon.technology/docs/category/test-history .  For example, you might need more space depending on the size of the blockchain state, and so on. 
3. We don't have explicit recommendations with AWS sizes.
4. All possible genesis flags can be found on our [CLI commands page](https://wiki.polygon.technology/docs/edge/get-started/cli-commands#genesis-flags). You can choose which flags (besides the ones used in our [setup guide](https://wiki.polygon.technology/docs/edge/get-started/set-up-ibft-locally)) would you need in your own setup. The flags `block-time` and `block-gas-limit` both have default values, which you can change if you'd like to (the default value can be found under each flag).
